tests/snappi_tests/intf_utils/intf_accept_metrics.py

```diff
@@ -0,0 +1,109 @@
+# This file defines the interfaces that snappi tests accept external metrics.
+
+#  Metrics data are organized into the hierarchies below
+#  ResourceMetrics
```

Comment:
I believe you are following the names of OTEL, but in our case, it will be better to be more specific to our use case.

So, we can rename the ResourceMetrics to TestMetrics.

Reply:
Good point! Done. Thank you!